project/Dependencies.scala

```diff
@@ -531,7 +531,7 @@ object Dependencies {
     "com.dimafeng" %% "testcontainers-scala-postgresql" % testContainersScalaV
   ) ++ slf4jBindingDependencies // During testing, add an slf4j binding for _all_ libraries.
 
-  val kindProjectorPlugin = "org.typelevel" %% "kind-projector" % kindProjectorV
+  val kindProjectorPlugin = "org.typelevel" % "kind-projector" % kindProjectorV cross CrossVersion.full
```

Comment:
😵

Reply:
yeah the [newer kind projector artifacts](https://mvnrepository.com/artifact/org.typelevel/kind-projector) depend on the *full* Scala version... took me a bit to figure out how to express that 🙂